src/digraph.jl

```diff
@@ -18,6 +18,26 @@ end
 
 DiGraph() = DiGraph(0)
 
+function DiGraph{T<:Real}(adjmx::SparseMatrixCSC{T})
+    dima, dimb = size(adjmx)
+    if dima != dimb
+        error("Adjacency / distance matrices must be square")
+    else
+        g = DiGraph(dima)
+        maxc = length(adjmx.colptr)
+        for c = 1:(maxc-1)
+            for rind = adjmx.colptr[c]:adjmx.colptr[c+1]-1
+                isnz = (adjmx.nzval[rind] > zero(T))
```

Comment:
maybe a != zero(T) to handle unexpected types?
Do we expect to get negative numbers here that wouldn't count as edges?